microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -404,4 +404,13 @@ Issued: 9 January 2025
 
 {product-title} release 4.16.29 is now available. The list of bug fixes that are included in the update is attached to the link:https://access.redhat.com/errata/RHBA-2025:0020[RHBA-2025:0020] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:0018[RHBA-2025:0018] advisory.
 
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-16-30-dp_{context}"]
+=== RHBA-2025:0142 - {microshift-short} 4.16.30 bug fix and enhancement advisory
+
+Issued: 15 January 2025
+
+{product-title} release 4.16.30 is now available. The list of bug fixes that are included in the update is attached to the link:https://access.redhat.com/errata/RHBA-2025:0142[RHBA-2025:0142] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:0140[RHSA-2025:0140] advisory.
```

Comment:
I've been out of the rel note space for a bit now but shouldn't the title also say it has security updates because the images advisory is an RHSA?

Reply:
We don't know; there's never been an engineering differentiation for the security updates to know if all of the OCP ones effect MicroShift.

Reply:
Oh gotcha! That makes sense!